packages/ra-ui-materialui/src/layout/LoadingIndicator.js

```diff
@@ -31,7 +32,7 @@ const LoadingIndicator = props => {
             {...rest}
         />
     ) : (
-        <RefreshIconButton />
+        <RefreshIconButton className={classNames(classes.icon, className)} />
```

Comment:
I'm afraid passing the same className as the progress component can have strange effects. Not sure how to handle this though

Reply:
The only solution to avoid possible breaking changes is to remove the classname passed to the refresh icon. This should not be a problem for users because as customisation of this icon was not possible before, they will only use the classes override.

Reply:
ok